Resources/Prototypes/_Sunrise/Entities/Objects/Fun/toys.yml

```
@@ -1115,3 +1115,123 @@
   - type: TriggerOnActivate
   - type: UseDelay
     delay: 1
+
+- type: entity
+  id: PlushieSeya
+  parent: BasePlushie
+  name: Plush Seya-Tanaka
+  description: An ordinary plush vulpkanin with dark fur, but it seems to be watching you.
+  components:
+  - type: Sprite
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)